pygotham/admin/__init__.py

```diff
@@ -29,6 +33,32 @@ def create_app(settings_override=None):
     """
     app = factory.create_app(__name__, __path__, settings_override)
 
+    @app.url_defaults
```

Comment:
Does this code match what we use elsewhere? Can we put it somewhere we can import it?